homework.md

```diff
@@ -4,83 +4,83 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language.
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+To style and layout web pages.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
+h1 {
 
 }
-```
+
 
 ## Question 7
 
 What is JavaScript?
-
-A programming language used on the internet
+Create dynamic and interactive web content like applications and browsers.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+There are three primary actions you can take when it comes to interacting with other developers’ code on GitHub:
+Fork, pull, Merge.
```

Comment:
Yes, these are actions you can do with Github.

Also, have a think about what problems Git & Github solve - for example, the problem of how do I save  versions of my work. Without a version control system like Git, I might save copies of a file as `myfile-v1.doc`, `myfile-v2.doc` etc. Git solves this problem by letting you save each version is in a separate commit. Can you think of other problems it solves as well? How about collaboration between developers?